src/vl-progress-bar.js

```diff
@@ -0,0 +1,93 @@
+import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+import '/node_modules/@govflanders/vl-ui-util/dist/js/util.js';
+import '/node_modules/@govflanders/vl-ui-progress-bar/dist/js/progress-bar.js';
+import '/node_modules/vl-ui-tooltip/dist/vl-tooltip.js';
+import './vl-progress-bar-item.js';
+
+/**
+ * VlProgressBar
+ * @class
+ * @classdesc Gebruik de progress-bar component om de vooruitgang te laten zien van een proces dat uit verschillende stappen bestaat.
+ * 
+ * @extends VlElement
+ * 
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-progress-bar/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-progress-bar/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-progress-bar.html|Demo}
+ * 
+ */
+export class VlProgressBar extends VlElement(HTMLElement) {
+    static get _observedChildClassAttributes() {
+        return ['numeric'];
+    }
+
+    constructor() {
+        super(`
+            <style>
+                @import '/src/style.css';
+                @import '/node_modules/vl-ui-tooltip/dist/style.css';
+            </style>
+            <div class="vl-progress-bar"></div>
+        `);
+    }
+
+    connectedCallback() {
+        this._processItems();
+        this._progressBar = new window['progress-bar'](this.constructor);
+    }
+
+    /**
+     * Zet een stap actief al dan niet met focus.
+     * 
+     * @param {number} number - Het nummer van de stap die actief wordt.
+     * @param {boolean} focus - Focus bepaalt of de stap in de progress bar focus krijgt.
+     */
+    updateStep(number, focus) {
```

Comment:
Je kan het eventueel ook zo maken dat een updateStep functie niet nodig is, maar dat de afnemer rechtstreeks op een item kan werken (`querySelector('#step-2').active = true; querySelector('#step-2').focus = true;`). Veranderingen die dan invloed hebben op de ganse progress bar kunnen met events terugvloeien naar de progress bar.
Of dat beter is dan met updateStep functie werken op de progress-bar, API-wise, laat ik ter discussie in de groep. 
Op die manier kan men in ieder geval ook via de DOM aanpassingen doorvoeren.

Je kunt het natuurlijk ook beide toelaten.

Reply:
Mijn visie is om zo dicht mogelijk bij de API van Webuniversum te blijven. Daar voorzien ze een updateStep, dus heb ik die overgenomen om hun API aan te houden. Een uitbreiding zoals voorgesteld kan toegepast worden maar lijkt me niet nodig. Het kan inderdaad nog intern besproken worden of de uitbreiding zinvol is om nu te voorzien.

Reply:
De API van Webuniversum is natuurlijk ook voor een stuk de API geworden omdat ze niet van de voordelen/gemakken van custom elements kunnen gebruikmaken.
Nu, het is in dit geval voor een groot stuk smaak, denk ik, dus tenzij anderen hetzelfde denken kunnen we het voor mij ook zo laten.

Reply:
gaat ge dan gene progress bar met 2 actieve elementen kunnen hebben als ge zelf als afnemer een step vastpakt en die ook actief zet? dus moet ge idd niet zoals bij radiogroup de rest "uit" zetten als ge ene "aan" zet?

Reply:
Ja, maar daarom dat ik zei dat "Veranderingen die dan invloed hebben op de ganse progress bar kunnen met events terugvloeien naar de progress bar."

Reply:
Lijkt me de introductie van de extra complexiteit niet waard.